### PR TITLE
Chore: add missing motionbuilder icon

### PR DESCRIPTION
**Pull request by moonyuet**
Add missing motionbuilder icon

### Testing Notes
1. Build the application addon with this branch
2. Make sure you include motionbuilder as part of the applications for your project
3. Go to AYON launcher
4. Moition Builder icon should be shown as expected

### CODE REVIEW

**Review by LiborBatek — APPROVED** [review]
LGTM

Screenshot 2024-07-05 161319

**Comment by iLLiCiTiT** [comment]
Please add it to `public/icons/` too.

**Comment by moonyuet** [comment]
> Please add it to `public/icons/` too.

done!

File: server/applications.json [hunk 1]
```diff
@@ -1192,6 +1192,19 @@
                     },
                     "arguments": {},
                     "environment": "{}"
+                },
+                {
```
Comment (iLLiCiTiT) [inline server/applications.json:1196]: ?

Comment (moonyuet) [reply server/applications.json]: my bad.

**Review by iLLiCiTiT — APPROVED** [review]

**Pull request merged**